src/main/kotlin/eu/europa/ec/eudi/sdjwt/SdJwtVerifier.kt

```diff
@@ -296,12 +288,12 @@ interface SdJwtVerifier<JWT> {
      * used to sign the SD-JWT
      * @param unverifiedSdJwt the SD-JWT to be verified
      * @return the verified SD-JWT, if valid. Otherwise, method could raise a [SdJwtVerificationException]
-     * The verified SD-JWT will contain a [JWT][SdJwt.Issuance.jwt] as both string and decoded payload
+     * The verified SD-JWT will contain a [JWT][SdJwt.jwt] as both string and decoded payload
      */
     suspend fun verifyIssuance(
```

Comment:
@babisRoutis I think this naming is no longer applicable. Maybe we should update the methods as follows:

* `suspend fun verify(jwtSignatureVerifier: JwtSignatureVerifier<JWT>, unverifiedSdJwt: String): Result<SdJwt<JWT>>` used to verify an SD-JWT
* `suspend fun verify(jwtSignatureVerifier: JwtSignatureVerifier<JWT>, keyBindingVerifier: KeyBindingVerifier<JWT>,, unverifiedSdJwt: String): Result<Pair<SdJwt<JWT>, JWT>>` used to verify an SD-JWT+KB

Similarly for methods that accept a JWS JsonObject.
Similarly for `SdJwtVcVerifier`.

This is also related to, and would also tackle, #293

Reply:
Agreed. Let's tackle this on another PR, addressing #293